habitat-lab/habitat/robots/stretch_robot.py

```diff
@@ -20,7 +20,7 @@ class StretchJointStates:
         [0.0] * 4 + [0.775, 0.0, -1.57000005, 0.0, -1.7375, -0.7125]
```

Comment:
```suggestion
        [0.0] * 4 + [0.775, 0.0, -np.pi/2, 0.0, -1.7375, -np.pi/4]
```

I forgot to update this here but had changed it [here](https://github.com/facebookresearch/home-robot/blob/c7ac74b4118083eb8e7146cf1a787aaca2cbe94a/src/home_robot/home_robot/motion/stretch.py#L55)

Reply:
I think we'll need `-np.pi/6` for -30 degrees.

Reply:
This is start of manipulation, where the agent needs to be looking further down

Reply:
My bad, meant to comment this on the navigation mode.

Reply:
Done